04/ht/calendar/calendar.js

```diff
@@ -0,0 +1,203 @@
+class Storage {
+  url = "https://myfirstproject-361208-default-rtdb.firebaseio.com/settingsCalendar/.json";
+  setData(value) {
+    return fetch(this.url, {
+      method: "PUT",
+      body: JSON.stringify(value),
+    })
+      .then((res) => res.json())
+      .catch((err) => catchResult(err));
```

Comment:
Можно и так оставить - чуточку короче
```suggestion
      .catch(catchResult);
```